### PR TITLE
Release VADF lifecycle management

### PR DESCRIPTION
**Pull request by doosuu**
# Description

## Azure DevOps PBI/Task reference

AB#12311

## Checklist

* [x] Vehicle App can be started with dapr run and is connecting to vehicle data broker
* [x] Vehicle App can process MQTT messages and call the seat service
* [x] Vehicle App can be deployed to local K3D and is running
* [ ] Created/updated tests, if necessary. Code Coverage percentage on new code shall be >= 70%.
* [ ] Extended the documentation in Velocitas repo
* [ ] Extended the documentation in README.md

* [x] Devcontainer can be opened successfully
* [x] Devcontainer can be opened successfully behind a corporate proxy
* [x] Devcontainer can be re-built successfully

* [x] Release workflow is passing

### CODE REVIEW

**Review by BjoernAtBosch — COMMENTED** [review]
Basically LGTM
Two findings

**Comment by BjoernAtBosch** [comment]
Tested on Ubuntu VM behind proxy

- [x] dev-container builds and runs (after deleting Docker's image cache and updating vscode to v1.75.0)
- [x] Starting Local VehicleApp runtime works (incl. Dapr installation)
- [x] Task "Local - VehicleApp (Dapr run)" works
- [ ] Task "CPP - Build and Run daprized App" is not working 

K3D setup not tested (yet)

File: .devcontainer/scripts/post-create.sh [hunk 1]
```diff
@@ -1,4 +1,5 @@
-#!/usr/bin/env bash
```
Comment (BjoernAtBosch) [inline .devcontainer/scripts/post-create.sh:1]: Script name should be in sync with Python template (there: postCreateCommand.h)

File: .velocitas.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            "name": "devenv-github-workflows",
+            "version": "cpp-template"
+        },
+        {
+            "name": "devenv-github-templates",
+            "version": "v1.0.0"
+        },
+        {
+            "name": "devenv-devcontainer-setup",
+            "version": "cpp-template"
```
Comment (BjoernAtBosch) [inline .velocitas.json]: Could that stay this way or do we need to release the changes in that repo first?

Comment (doosuu) [reply .velocitas.json]: We need to release the changes in that repo first, I will provide a PR.

File: .devcontainer/scripts/post-create.sh (hunk 1 above)
Comment (doosuu) [reply .devcontainer/scripts/post-create.sh:1]: Done

File: .velocitas.json (hunk 2 above)
Comment (BjoernAtBosch) [reply .velocitas.json]: Fixed

File: .devcontainer/scripts/post-create.sh (hunk 1 above)
Comment (BjoernAtBosch) [reply .devcontainer/scripts/post-create.sh:1]: &#x1F44D;

**Review by BjoernAtBosch — APPROVED** [review]
LGTM

**Pull request merged**